### PR TITLE
[sdn_tests]: Added binding infra to pins_ondatra

### PR DESCRIPTION
**Pull request by VSuryaprasad-HCL**
### Description of PR
[sdn_tests]: Added binding infra to pins_ondatra.

Summary:
- Adding Bindings and GNOI Infra-code.

Build Result:

       INFO: From Generating Descriptor Set proto_library @com_github_openconfig_gnsi//pathz:pathz_proto:
       github.com/openconfig/gnsi/pathz/authorization.proto:43:1: warning: Import 
       github.com/openconfig/gnsi/version/version.proto is unused.
       github.com/openconfig/gnsi/pathz/pathz.proto:25:1: 
       warning: Import github.com/openconfig/gnsi/version/version.proto is unused.
       INFO: Elapsed time: 1240.892s, Critical Path: 289.51s
       INFO: 975 processes: 251 internal, 724 linux-sandbox.
       INFO: Build completed successfully, 975 total actions 

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [-] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311

### CODE REVIEW

**Review by rkavitha-hcl — APPROVED** [review]

File: sdn_tests/pins-ondatra/infrastructure/testhelper/gnoi.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+		return gnmi.Get(t, d, gnmi.OC().System().BootTime().State())
+	}
+)
+
+// RebootParams specify the reboot parameters used by the Reboot API.
+type RebootParams struct {
+	request       any
+	waitTime      time.Duration
+	checkInterval time.Duration
+	lmTTkrID      string // latency measurement testtracker UUID
```
Comment (parth-opensrc) [inline sdn_tests/pins-ondatra/infrastructure/testhelper/gnoi.go:39]: remove.

File: sdn_tests/pins-ondatra/infrastructure/testhelper/gnoi.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+	lmTitle       string // latency measurement title
```
Comment (parth-opensrc) [inline sdn_tests/pins-ondatra/infrastructure/testhelper/gnoi.go:40]: remove

**Comment by parth-opensrc** [comment]
- Change desc to `[sdn_tests]: Added binding infra to pins_ondatra.`
- Please add `testhelper` code in separate PRs.

File: sdn_tests/pins-ondatra/infrastructure/testhelper/gnoi.go (hunk 2 above)
Comment (VSuryaprasad-HCL) [reply sdn_tests/pins-ondatra/infrastructure/testhelper/gnoi.go]: removed a per review comments.

**Pull request closed without merging**